index.js

```diff
@@ -0,0 +1,59 @@
+
+function calculate (input) {
+    //numberfy stringy numbers
+    for (let i=0; i < input.length; i++) {
+        const element = input[i];      
+        //check for strings
+        if ((typeof(element)) === 'string') {
+            //check for numbers in those strings
+            if (!isNaN(element)) {
+                // change stringy numbers to regular numbers
+                input[i] = parseInt(element);
+            }
+        }
+    }
+   
+    // removal of undefined and non-numerics 
+    // use filter() to return numbers and operators
+
+    function getNumsAndOperators(element){
+        if (element === '') {
+            return false;
+        }
+        return !isNaN(element) || element === '+' || element === '-' || element === '*' || element === '/'; 
+    }
+    input = input.filter(getNumsAndOperators);
+
+
+    // returns NaN if the array does not represent a proper operation
+
+
+    // how input should run
+    const val1 = input[0];
+    const operand = input[1];
+    const val2 = input[2];
+
+    switch(operand) {
+        case '+':
+        return (val1 + val2); 
+        break;
```

Comment:
There is no need to include a break statement after a return statement such as what you have here in any scenario, as the return statement itself acts as a break. Therefore, the break statements that you've included after your return statements in each of your switch cases will never be reached/read.